### ISSUE TITLE
User auth is not working

### ISSUE THREAD

**Issue by Uthiol** [Teekeks/pyTwitchAPI#87]
Hello,

I've been trying to make a User authentification using this code : 

```
twitch = Twitch(clientId, authenticate_app=False)
target_scope = [AuthScope.USER_READ_FOLLOWS]
auth = UserAuthenticator(twitch, target_scope, force_verify=False)
token, refresh_token = auth.authenticate()
twitch.set_user_authentication(token, target_scope, refresh_token)
```

the line 
`token, refresh_token = auth.authenticate()`
crashes saying 
`KeyError: 'access_token'`

I've tried on 2.3.2 and 2.4.1 and it didn't worked

Is there anything i'm missing that can cause this issue ? 

Thanks a lot for your work and help !

**Comment by Teekeks** [comment]
If you update to the newest version I just released (v2.4.2) you should get a more meaningfull error message of what is wrong. I am guessing that the authentication failed for some reason.

**Comment by Uthiol** [comment]
unfortunately it doesn't change anything 

![image](https://user-images.githubusercontent.com/45762766/134026128-26c3a7d2-d15b-4776-9c3e-49a3b1e485eb.png)

As you can see. 

But i've took the method on local and it seems that the server is answering "missing client secret"

![image](https://user-images.githubusercontent.com/45762766/134027377-e78eca1a-7b22-4cd7-a20c-bfcb344189d6.png)

that seems logical since the oauth2/token needs client secret

did this part of the code changed a while ago ?
